templates/help/faq.hbs

```diff
@@ -8,6 +8,30 @@
     }}
     {{! Access the page context with `page` (see globals.json)}}
     <h2>{{page.title}}</h2>
+    {{!This is for the collapsible lists}}
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+        {{#each sections.help.questions}}
+            <div class="panel panel-default">
+                <div class="panel-heading" role="tab" id={{@index}}: {{this}}>
```

Comment:
This isn't how you should give an id see http://www.w3schools.com/tags/att_global_id.asp